ts/form/package.json

```
@@ -5,7 +5,7 @@
   "main": "index.js",
   "module": "index.js",
   "type": "module",
-  "repository": "https://github.com/vaadin/fusion/frontend/packages/form",
+  "repository": "https://github.com/vaadin/fusion/ts/form",
```

This is not a valid way to define a repository with subdirectory for the package. See https://docs.npmjs.com/cli/v6/configuring-npm/package-json#repository

It should be like this (if using the `ts/form` dir):
```suggestion
  "repository": {
    "type" : "git",
    "url" : "https://github.com/vaadin/fusion.git",
    "directory": "ts/form"
  },
```

---

Didn't know that, thanks!